### PR TITLE
docs: add vercel logo

### PR DESCRIPTION
**Pull request by anikethsaha**
vercel is sponsoring our pro account. 
This is to add their logo in our docs and readme

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -30,3 +30,10 @@ Please consider donating if you think docsify is helpful to you or that my work
 ## Community
 
 Users and the development team are usually in the [Gitter chat room](https://gitter.im/docsifyjs/Lobby).
```
Comment (sy-records) [inline docs/README.md:32]: Should we change `Gitter` to `Discord`?

Comment (Koooooo-7) [reply docs/README.md:32]: @sy-records  I remembered that @trusktr updated a new README.md, I thing it will be changed later.

Comment (sy-records) [reply docs/README.md:32]: #1224

**Comment by anikethsaha** [comment]
> Instead of embedding the SVG code, let's re-use the SVG file.

yea make sense

**Pull request merged**